src/Stratis.Bitcoin/Base/Deployments/ThresholdConditionCache.cs

```diff
@@ -9,18 +9,18 @@ namespace Stratis.Bitcoin.Base.Deployments
     public class ThresholdConditionCache
     {
         /** What block version to use for new blocks (pre versionbits) */
-        public static int VersionbitsLastOldBlockVersion = 4;
+        private static int VersionbitsLastOldBlockVersion = 4;
```

Comment:
Looks like public methods that are not used now made private. Not sure if that's correct. 
Some PRs for example introduce some public fields and use them only in follow up PRs.

Reply:
yeah I made those private because they're not used outside, or not used.
I appreciate that these might be used in the future so I left them there, just made them private.